### ISSUE TITLE
Accessibility issue with the Carousel component

### ISSUE THREAD

**Issue by wingl2** [carbon-design-system/carbon-for-ibm-dotcom#12232]
### Description

Our DPH UI is using the `@carbon/ibmdotcom-web-components/es/components-react/carousel/carousel` component to show a list of clickable tiles.

When these tiles are scrolled off screen, the `carousel` component will tag them with `aria-hidden=true`.
However, our tiles are clickable and in the tabbing focus sequence.

When scanning with `IBM Equal Access Accessibility Checker`, it shows the violations in the screenshot.
Are there any work around on this?  or can the offscreen tiles be set to display: none?

![Image](https://github.com/user-attachments/assets/4cfdc013-d2fa-4569-a274-9b1f54d00abf)

### Component(s) impacted

const DDSCarousel = loadable(
	() => import("@carbon/ibmdotcom-web-components/es/components-react/carousel/carousel"),
	{ ssr: false }
);
...
return (
	<DDSCarousel
		nextButtonText={intl.formatMessage(messages.nextButtonText)}
		prevButtonText={intl.formatMessage(messages.prevButtonText)}
	>
		{dataProductTiles}
	</DDSCarousel>
);

### Browser

Chrome

### Carbon for IBM.com version

2.17.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

https://dataplatform.dev.cloud.ibm.com/dpx/?context=dph

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

None

### Steps to reproduce the issue (if applicable)

1. Navigate to https://dataplatform.dev.cloud.ibm.com/dpx/?context=dph
2. Scan the DPH homepage with IBM accessibility checker

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
